src/conn/conn_layered.c

```diff
@@ -356,7 +356,7 @@ __disagg_pick_up_checkpoint(WT_SESSION_IMPL *session, uint64_t meta_lsn)
     /* We should not pick up a checkpoint with an earlier LSN. */
     WT_ACQUIRE_READ(current_meta_lsn, conn->disaggregated_storage.last_checkpoint_meta_lsn);
     if (meta_lsn < current_meta_lsn)
-        WT_RET_MSG(session, EINVAL,
+        WT_RET_MSG(session, WT_ERROR_LOG_ADD(EINVAL),
```

Comment:
nit: [here](https://github.com/wiredtiger/wiredtiger/blob/3c88dc21e555da10e34ed274bd1ea0053446bef6/src/block/block_open.c#L380) could be some better examples where to use this.

There are a few occurrences of `ENOENT` in the file and it might be unclear which one resulted in an error. Other instances of `ret = X` are also good candidates.